stack.cabal

```diff
@@ -49,6 +49,10 @@ flag static
   -- Not intended for general use. Simply makes it easier to
   -- build a fully static binary on Linux platforms that enable it.
 
+flag use-old-locale
+  default:     False
+  description: Use old-locale instead of time >= 1.5
```

Comment:
Sorry again for the confusion in #2377.

There isn't much point in increasing the complexity of the cabal file only to make `stack` compatible with `lts-2` (of course `stack` then still needs _lots_ of extra-deps).

To build stack with ghc-7.8 you can already use `stack install --stack-yaml stack-7.8.yaml`.